view/frontend/web/js/view/delivery-options.js

```diff
@@ -228,11 +238,17 @@ define(
 
         checkout.convertDeliveryOptionsToShippingMethod(event.detail, {
           onSuccess: function(response) {
-            if (!response.length) {
+            if (! response.length) {
               return;
             }
 
-            quote.shippingMethod(deliveryOptions.getNewShippingMethod(response[0].element_id));
+            try {
+              if (quote.shippingMethod()) {
```

Comment:
Dit is heel gek, je hebt niks aan deze `try` als je geen `catch` gebruikt, `finally` gebeurt sowieso dus dat kan weg en de code daarin kan er gewoon los onder.

Reply:
de try is alleen om te zorgen dat execution door gaat, ik zal er een lege catch achter zetten, en even opnieuw testen...